include/zephyr/net/wifi_nm.h

```diff
@@ -33,7 +33,7 @@ extern "C" {
 /** Types of Wi-Fi interface */
 enum wifi_nm_iface_type {
 	/** IEEE 802.11 Wi-Fi Station */
-	WIFI_TYPE_STA = 0,
+	WIFI_TYPE_STA = 1,
```

Comment:
please create a separate commit for this (FYI,  I fixed this locally as well along with few other changes).